class ApiError(Exception):
    """Исключение, используемое для обработки ошибок при работе с API."""

    pass
